README.md

```diff
@@ -223,6 +225,13 @@ pd.DataFrame(res['projectBySlug'], index=[0])
 0            ETH  0x7c5a0ce9267ed19b22f8cae653f198e3e8daf098  Santiment  santiment    SAN  https://twitter.com/santimentfeed
 ```
 
+### Rate Limit Tools
+
+There are two functions, which can help you in handling the rate limits:
+* ``is_rate_limit_exception`` - Returns whether the exception caught is because of rate limitation
+* ``rate_limit_time_left`` - Returns the time left before the rate limit expires
```

Comment:
Can you also add a short code example with a `try` and catch clauses that use these functions?